.github/ISSUE_TEMPLATE/todo.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Todo
+about: Create simple todo for this project
+title: ''
+labels: ''
+assignees: ''
```

Comment:
What is the purposed of these compared to using GitHub assignees and labels?

Reply:
Those are GitHub assignees and labels. They get auto-assigned on issue creation. These are generated via GitHub, ref.: https://help.github.com/en/articles/about-issue-and-pull-request-templates